### PR TITLE
Mangalib Update

### PR DESCRIPTION
**Pull request by Mirazex**
fix & added new info (bcs website updated)

### CODE REVIEW

File: websites/M/Mangalib/dist/metadata.json [hunk 1]
```diff
@@ -6,18 +6,13 @@
   "service": "Mangalib",
   "description": {
     "ru": "Сайт для прочтения манги на русском языке.",
-    "en": "Website for reading manga"
+    "en": "Website for reading manga on russian language"
```
Comment (Bas950) [inline websites/M/Mangalib/dist/metadata.json:9]: revert that, or make it `Website for reading manga in Russian`

File: websites/M/Mangalib/dist/metadata.json [hunk 2]
```diff
@@ -6,18 +6,13 @@
...
-  "thumbnail": "https://i.imgur.com/6tzn5Gc.png",
-  "color": "#E48F13",
-  "tags": [
-    "manga",
-    "mangalib",
-    "read",
-    "anime"
-  ],
+  "version": "1.3.0",
+  "logo": "https://mangalib.me/icons/android-icon-144x144.png",
```
Comment (Bas950) [inline websites/M/Mangalib/dist/metadata.json:13]: upload to imgur

File: websites/M/Mangalib/tsconfig.json [hunk 3]
```diff
@@ -1,5 +1,5 @@
 {
-  "extends": "../../../tsconfig.json",
+  "extends": "../tsconfig.json",
   "compilerOptions": {
     "outDir": "./dist/"
   }
```
Comment (Bas950) [inline websites/M/Mangalib/tsconfig.json:5]: revert

**Comment by Bas950** [comment]
Also upload proof of it working and update your fork

**Comment by Mirazex** [comment]
jPgdQcbXxY

**Comment by Mirazex** [comment]
and add other website (Ranobelib)
VK3QSOCh5c

**Comment by Mirazex** [comment]
and another one (Hentailib)
4gsHRXGnbE

File: websites/M/Mangalib/tsconfig.json [hunk 4]
```diff
@@ -1,5 +1,5 @@
 {
-  "extends": "../../../tsconfig.json",
+  "extends": "../../../ tsconfig.json",
   "compilerOptions": {
     "outDir": "./dist/"
   }
```
Comment (Bas950) [inline websites/M/Mangalib/tsconfig.json:5]: remove the space after the /

File: websites/H/Hentailib/tsconfig.json [hunk 5]
```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "../../../ tsconfig.json",
+  "compilerOptions": {
+    "outDir": "./dist/"
+  }
+}
```
Comment (Bas950) [inline websites/H/Hentailib/tsconfig.json:6]: remove the space after the /

File: websites/R/Ranobelib/tsconfig.json [hunk 6]
```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "../../../ tsconfig.json",
+  "compilerOptions": {
+    "outDir": "./dist/"
+  }
+}
```
Comment (Bas950) [inline websites/R/Ranobelib/tsconfig.json:6]: remove the space after the /

File: websites/R/Ranobelib/dist/metadata.json [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+    "name": "Mirax",
+    "id": "223355525826347018"
+  },
+  "service": "Ranobelib",
+  "description": {
+    "ru": "Сайт для прочтения ранобэ на русском языке.",
+    "en": "Website for reading ranobe in Russian"
+  },
+  "url": "ranobelib.me",
+  "version": "1.3.0",
```
Comment (Bas950) [inline websites/R/Ranobelib/dist/metadata.json:12]: version 1.0.0 on release

**Pull request merged**